provisioning/provisioning-service-client/src/test/java/com/microsoft/azure/sdk/iot/provisioning/service/configs/TpmAttestationTest.java

```diff
@@ -18,28 +18,6 @@
     private static final String VALID_ENDORSEMENT_KEY = "validEndorsementKey";
     private static final String VALID_STORAGE_ROOT_KEY = "validStorageRootKey";
 
-    /* SRS_TPM_ATTESTATION_21_001: [The constructor shall throw IllegalArgumentException if the provided endorsementKey is null or empty.] */
```

Comment:
I removed some unnecessary null/empty checks, so a lot of unit tests needed to be deleted

Reply:
could you add some context to why these null/empty checks were removed? is it because those values would only be set by service and hence would never be null/empty?

Reply:
It was a mix of fields where the service would validate them for us and readonly fields that the service wouldn't return null values for